paper/paper.bib

```diff
@@ -0,0 +1,20 @@
+@book{Lawler:1995,
```

Comment:
@sadrasabouri I'm not sure about this citation format.
Where did you get this?

Reply:
Yes that was not OK
I fetch the right one from Google Scholar.